homeassistant/components/homekit/type_lights.py

```diff
@@ -0,0 +1,209 @@
+"""Class to hold all light accessories."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_RGB_COLOR, ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS, SUPPORT_RGB_COLOR)
+from homeassistant.const import ATTR_SUPPORTED_FEATURES, STATE_ON, STATE_OFF
+
+from . import TYPES
+from .accessories import HomeAccessory, add_preload_service
+from .const import (
+    CATEGORY_LIGHT, SERV_LIGHTBULB,
+    CHAR_BRIGHTNESS, CHAR_HUE, CHAR_ON, CHAR_SATURATION)
+
+_LOGGER = logging.getLogger(__name__)
+
+RGB_COLOR = 'rgb_color'
+
+
+class Color:
+    """Class to handle color conversions."""
+
+    # pylint: disable=invalid-name
+
+    def __init__(self):
+        """Initialize a new Color object."""
+        self.hue = None         # [0, 360]
+        self.saturation = None  # [0, 1]
+
+    def calc_hsv_to_rgb(self):
```

Comment:
I don't know what the exact status is of the migration to hue/saturation as the core color model. (see #11288)
But can't that already be used here @armills
Then you only have to divide saturation by 100, don't need a class for that ;)

Reply:
Year, that would make thinks a lot easier certainly, but until the PR gets merged this should work.

Reply:
Planning to get #11288 in shape this weekend. Got all the dependencies in place, just need to go through and resolve all the merge conflicts.